tests/acceptance/31_testall_selftest/flaky_fail.cf

```diff
@@ -0,0 +1,5 @@
+bundle agent main
+{
+  reports:
+    "$(this.promise_filename) FLAKEY";
```

Comment:
Shouldn't this have a ticket number? Also, doesn't it mean there will always be at least one flakey test?

Reply:
Right. I was going to move these tests to another mechanism maybe in Makefile.am to just check the results since there are currently THREE "failures" that would break the build but I would like to test.